nutanix/resource_nutanix_virtual_machine.go

```diff
@@ -981,13 +981,15 @@ func resourceNutanixVirtualMachineRead(d *schema.ResourceData, meta interface{})
 
 func resourceNutanixVirtualMachineUpdate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*Client).API
+	var hotPlugChange = true
 
 	log.Printf("[Debug] Updating VM values %s", d.Id())
 
 	//First, shutDown the VM.
```

Comment:
Could you remove these commented lines?

Reply:
Sure, It's done.
I've also remove some others that weren't useful.
Thanks

Reply:
Thanks!